app/server/smtp.js

```diff
@@ -1,15 +0,0 @@
-Meteor.startup(function () {
```

Comment:
Should we re-open this PR, and cherry-pick your commits, so this info doesn't appear in the commit history?

Reply:
@brylie No, that won't be necessary. I changed my password.